04-mpu6050/scripts/tempout.sh

```diff
@@ -11,4 +11,11 @@ while :
 do
 	RES_H=$(i2cget -y 0 0x68 $TEMP_OUT_H)
 	RES_L=$(i2cget -y 0 0x68 $TEMP_OUT_L)
+
+	RES_H=$((RES_H << 8))
+	RES_H=$((RES_H & 0xff00))
```

Comment:
This & 0xff00 is not required. After left shift by 8 bits 0-7 will have value 0.

Reply:
Fixed that